package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "elasticio-sailor-nodejs",
   "description": "The official elastic.io library for bootstrapping and executing for Node.js connectors",
-  "version": "2.6.24",
+  "version": "2.6.24-consumer-with-reconnect",
```

Comment:
Version should be increased by 1 at patch number `2.6.25-consumer-with-reconnect`

Reply:
and the final version should be 2.6.25 😉